src/components/Data/Collections/Create.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div>
+    <create-or-update
+      headline="Create collection"
+      @collection-create::create="create"
+      :index="index">
+    </create-or-update>
+  </div>
+</template>
+
+<script>
+  import CreateOrUpdate from './CreateOrUpdate'
+  import { fetchCollectionDetail, resetCollectionDetail, createCollection } from '../../../vuex/modules/collection/actions'
+  import { collectionName } from '../../../vuex/modules/collection/getters'
+  import { indexesAndCollections } from '../../../vuex/modules/data/getters'
+
+  export default {
+    name: 'CollectionCreate',
+    props: {
+      index: String
+    },
+    components: {
+      CreateOrUpdate
+    },
+    vuex: {
+      actions: {
+        fetchCollectionDetail,
+        resetCollectionDetail,
+        createCollection
+      },
+      getters: {
+        collectionName,
+        indexesAndCollections
+      }
+    },
```

Comment:
Because when a collection is created we have to add it in the list of already existing collection in left column. When we update the collection, we have to do nothing on store